src/algorithms/policy_gradient/ppo.jl

```diff
@@ -246,18 +246,19 @@ function _update!(p::PPOPolicy, t::AbstractTrajectory)
                 @error "TODO:"
             end
             s = send_to_device(D, select_last_dim(states_flatten, inds))  # !!! performance critical
-            a = vec(actions)[inds]
```

Comment:
Is there any reason why `a` is the only one not run through `send_to_device`?

Reply:
That's due to a historical reason. I think in the current version of CUDA.jl, it's safe to call `send_to_device` here.